docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,443 @@
+- You need a folder for each CA, such as `/tls-ca` or `/org1-ca` or `/int-ca`. This folder can be under the Fabric CA client or elsewhere where the CA client can access the path.
+- Under each CA folder, you need a folder to store the TLS certificate, such as `org1-ca/crypto`. This TLS certificate secures the communications between the CA client and the CA server. Secondly, you need to create a folder to place the generated certificates on an enroll command. If this is an admin identity, it would be  `org1-ca/admin`.
+
+You can also use environment variables to specify certificate locations.
+- `FABRIC_CA_CLIENT_TLS_CERTFILES` - Specify the location and name of the TLS certificate. If the path of the environment variable FABRIC_CA_CLIENT_TLS_CERTFILES is not an absolute path, it will be parsed as relative to the client’s home directory.
+- `FABRIC_CA_CLIENT_HOME` - Specify the path to where the generated certificates should be stored. This path is relative to the Fabric CA client binary.
+- `FABRIC_CA_CLIENT_MSPDIR` - While you can use this environment variable to specify the name of the folder under `FABRIC_CA_CLIENT_HOME` where the generated certificates are stored, _a better option if the certificates are not being stored in the default `msp` folder is to pass the `-M` flag on the enroll command itself and explicitly specify the location for the generated certificates._ If not specified on the command, the location defaults to `$FABRIC_CA_CLIENT_HOME/msp`. **Important:** When multiple members are enrolled with the CA, you need to store their certificates in distinct folders.
+
+The first time you issue an `enroll` command from the CA client, the `fabric-ca-client-config.yaml` is generated in the `$FABRIC_CA_CLIENT_HOME` directory, if one is not already there. When you customize the values in this file, they can be used automatically by the CA client and do not have to be passed on the command line on a subsequent `enroll` command.
+
+## Submitting transactions from the CLI
+
+Two sets of CLI commands are included with the CA server and CA client binary files:
+
+- Use the [Fabric CA **server** CLI commands](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/servercli.html) to deploy and update the CA server.
+
+- Use the [Fabric CA **client** CLI commands ](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html) to submit requests to your CA server after it is set up, such as registering, enrolling or revoking identities.
+
+We will use these CLI commands throughout this topic.
+
+## Deploy the TLS CA
+
+Regardless of whether you are setting up a TLS CA, an organization CA or an intermediate CA, the process follows the same overall steps. The differences will be in the modifications you make to the CA server configuration .yaml file. The following instructions describe the process beginning with the TLS CA.
+
```

Comment:
It might be good to have a higher level list like this. 
- Deploy TLS CA
- Deploy Root CA
- Deploy Int CA

Reply:
I created a new section for this.